koku/api/user_settings/test/test_views.py

```diff
@@ -135,3 +136,75 @@ def test_no_userset_cost_type(self):
 
         data = response.data
         self.assertEqual(data.get("data"), COST_TYPES)
+
+
+class AccountSettingsViewTestRBACTest(IamTestCase):
+    def setUp(self):
+        """Set up the account settings view tests."""
+        super().setUp()
+        self.client = APIClient()
+        self.base_url = reverse("account-settings")
+        self.account_settings = {"currency": "USD", "cost-type": "blended_cost"}
+
+    no_access = {"aws.account": {"read": ["*"]}, "aws.organizational_unit": {"read": ["*"]}}
+    read = {"settings": {"read": ["*"]}}
+    write = {"settings": {"write": ["*"]}}
+    read_write = {"settings": {"read": ["*"], "write": ["*"]}}
+
+    # deprecated permissions
+    @RbacPermissions(no_access)
+    def test_no_access_to_get_request(self):
+        for acct_setting in self.account_settings.keys():
```

Comment:
Keys are iterated over by default. There's no need to get the keys view (and it saves a couple of calls).
```suggestion
        for acct_setting in self.account_settings:
```